js/webform_civicrm_admin.js

```diff
@@ -580,18 +580,5 @@ var wfCiviAdmin = (function ($, D) {
     }
   };
 
-  /**
-   * This block uses CiviCRM's jQuery not Drupal's version
-   * TODO: Move more code here! Drupal's version of jQuery is ancient.
```

Comment:
Glad we didn't ever do that TODO. Easier to get rid of this block completely.

Reply:
:-) yes phew!